.gitignore

```diff
@@ -246,3 +246,4 @@ javascript/new-src/node_modules
 .cproject
 .settings
 .vscode/
+OLA.*
```

Comment:
I think you said this was for some IDE, can you make it more strict please, isn't OLA.* just based on your project name, so it won't actually work for other people?

Reply:
If I'm not mistaken OLA is the name that gets autogenerated when importing the project from existing source.
I'll change it to be specifically the 4 files generated?

Reply:
Changed.